insalan/tournament/admin.py

```diff
@@ -856,3 +863,15 @@ def launch_swiss_matchs_action(self,request,queryset):
         self.message_user(request,_("Les matchs ont bien été lancés"))
 
 admin.site.register(SwissMatch, SwissMatchAdmin)
+
+
+class SeatingAdmin(admin.ModelAdmin):
+    """Admin handler for Seating"""
+
+    list_display = ("id", "event", "tournament", "team")
+    search_fields = ["event", "tournament", "team"]
+
+    list_filter = ["tournament", "tournament__event", "tournament__game"]
```

Comment:
You should not be using tournament__event but instead just event. The tournament field is intended to be only a tournament belonging to the event stored in the model.